apps/wallet-dashboard/components/Dialogs/Assets/views/DetailsView.tsx

```diff
@@ -0,0 +1,190 @@
+                                                {nftMeta?.projectUrl}
+                                            </Link>
+                                        }
+                                        fullwidth
+                                    />
+                                )}
+                                {nftMeta?.creator && (
+                                    <KeyValueInfo
+                                        keyText="Creator"
+                                        value={nftMeta?.creator ?? '-'}
+                                        fullwidth
+                                    />
+                                )}
+                            </div>
+                        )}
+
+                        <Collapsible defaultOpen title="Details">
+                            <div className="flex flex-col gap-xs px-md pb-xs pt-sm">
+                                {ownerAddress && (
+                                    <KeyValueInfo
+                                        keyText="Owner"
+                                        value={
+                                            <ExplorerLink
+                                                linkProps={{ type: ExplorerLinkType.Address }}
+                                                address={ownerAddress}
+                                            >
+                                                {formatAddress(ownerAddress)}
+                                            </ExplorerLink>
+                                        }
+                                        fullwidth
+                                    />
+                                )}
+                                {objectId && (
+                                    <KeyValueInfo
+                                        keyText="Object ID"
+                                        value={formatAddress(objectId)}
+                                        fullwidth
+                                    />
+                                )}
+                            </div>
+                        </Collapsible>
+                        {metaKeys.length ? (
+                            <Collapsible defaultOpen title="Attributes">
+                                <div className="flex flex-col gap-xs px-md pb-xs pt-sm">
+                                    {metaKeys.map((aKey, idx) => {
+                                        const { value, valueLink } = formatMetaValue(
+                                            metaValues[idx],
+                                        );
+                                        return (
+                                            <KeyValueInfo
+                                                key={idx}
+                                                keyText={aKey}
+                                                value={
+                                                    <Link key={aKey} href={valueLink || ''}>
```

Comment:
not all of them should be links, for example,  the description and name shouldn't be clickable :see_no_evil:. when you hover over the description, you can see it's pointing to `/assets`
![image](https://github.com/user-attachments/assets/be70ca71-9d7d-45f7-aba5-39e8d163c4d6)

Reply:
also the links should have a different color depending on the mode (light/dark mode) like in figma :pray: 
![image](https://github.com/user-attachments/assets/b271cf98-359f-4002-8119-29dc67cace1b)